groups/data-lake/main.tf

```diff
@@ -14,6 +10,24 @@ data "aws_security_group" "mongo_db" {
   }
 }
 
+data "vault_generic_secret" "secrets" {
+  path = "applications/${var.aws_profile}/data-lake"
+}
+
+# ------------------------------------------------------------------------------
+# Locals
+# ------------------------------------------------------------------------------
```

Comment:
Can we drop the separator comments like this from the file? The 'Locals' comment here is followed by a `locals` block and the `Resources` comment later in the file is followed immediately by a `resource` so should be self explanatory, and we always have the option to split files code into separate files if some more separation is needed.

Reply:
Fair point. It was more for visual separation given how large the initial implementation of `main.tf`